spec/models/person_spec.rb

```diff
@@ -233,6 +237,42 @@
     end
   end
 
+  describe 'calculate_title_order' do
+    let!(:director) {create :person, title: 'Director'}
```

Comment:
There's no harm in having this in now, but it would need to come back out as we move titles into the database.